diff --git a/src/ui/include/mainwindow.h b/src/ui/include/mainwindow.h
index 299e026..741c404 100644
--- a/src/ui/include/mainwindow.h
+++ b/src/ui/include/mainwindow.h
@@ -22,6 +22,10 @@ namespace Ui {
 class MainWindow;
 }
 
+#ifndef  NOTEPADQQ_UNUSED
+# define NOTEPADQQ_UNUSED(x) ((void)(x))
+#endif
+
 class MainWindow : public QMainWindow
 {
     Q_OBJECT
@@ -34,6 +38,13 @@ public:
     static QList<MainWindow *> instances();
     static MainWindow * lastActiveInstance();
 
+    // GH #792: Need to press TAB twice to register one TAB in editor
+    // Also see https://stackoverflow.com/a/21351638
+    virtual bool focusNextPrevChild(bool next) {
+        NOTEPADQQ_UNUSED(next);
+        return false;
+    }
+
     /**
      * Describes the result of a tab closing process.
      */
